typescript/src/README.md

```diff
@@ -351,7 +351,56 @@ console.log(`Transaction confirmed! Explorer link: https://sepolia.basescan.org/
 
 #### Solana
 
-For Solana, we recommend using the `@solana/web3.js` library to send transactions. See the [examples](https://github.com/coinbase/cdp-sdk/tree/main/examples/typescript/solana/signAndSendTransaction.ts).
+You can use CDP SDK to send transactions on Solana.
+
+For complete examples, check out [sendTransaction.ts](https://github.com/coinbase/cdp-sdk/blob/main/examples/typescript/solana/sendTransaction.ts), [sendManyTransactions.ts](https://github.com/coinbase/cdp-sdk/blob/main/examples/typescript/solana/sendManyTransactions.ts), and [sendManyBatchedTransactions.ts](https://github.com/coinbase/cdp-sdk/blob/main/examples/typescript/solana/sendManyBatchedTransactions.ts).
+
+```typescript
```

Comment:
nit: could you please run the formatter on the TS code in this snippet so that it is formatted with 2 white spaces instead of 4. Easiest way to do this would probably be to copy-paste the snippet into an actual TS file in the src folder, run the formatter, then copy-paste the formatted result back in here